src/core/CMakeLists.txt

```diff
@@ -176,6 +176,7 @@ set(COMMON_SOURCES
     thread/announce_begin_server.cpp
     thread/announce_sender.cpp
     thread/child_table.cpp
+    thread/dua_manager.cpp
```

Comment:
Shall we also add this in `Andriod.mk` and `BUILD.gn`?

Reply:
added